UPGRADE-2.0.md

```diff
@@ -1,57 +1,97 @@
-  choice of setting an HTTP client supporting syncronous, asyncronous or both
-  transport methods.
+  in the scope.
+
+- The `curl_method` option has been removed.
+
 - The `curl_path` option has been removed.
+
 - The `curl_ipv4` option has been removed.
+
 - The `curl_ssl_version` option has been removed.
+
 - The `verify_ssl` option has been removed.
+
 - The `ca_cert` option has been removed.
+
 - The `proxy` option has been removed in favour of leaving to the user the burden
   of configuring the HTTP client options using a custom client.
-- The `processors` option has been removed in favour of leaving to the user the
-  choice of which processors add or remove using the appropriate methods of the
-  `Client` and `ClientBuilder` classes.
-- The `processors_options` option has been removed in favour of leaving to the
-  user the burden of adding an already configured processor instance to the client.
+
+- The `processors` option has been removed.
+
+- The `processors_options` option has been removed.
+
 - The `transport` option has been removed in favour of setting it using the
   client builder.
+
 - The `install_default_breadcrumb_handlers` option has been removed.
-- The `http_client_options` has been added to set the options that applies to the
-  HTTP client chosen by the user as underlying transport method.
-- The `open_timeout` option has been added to set the maximum number of seconds
-  to wait for the server connection to open.
-- The `excluded_loggers` option has been added to set the list of logger 'progname's
-  to exclude from breadcrumbs.
-- The `environments` option has been added to set the whitelist of environments
-  that will send notifications to Sentry.
+
 - The `serialize_all_object` option has been added to configure whether all the
   object instances should be serialized.
+
 - The `context_lines` option has been added to configure the number of lines of
   code context to capture.
+
 - The `server` option has been renamed to `dsn`.
 
-### Client
+### Misc
+
+- All the classes have been renamed and moved around to follow the PSR-4
+  convention and `final` have been added where appropriate.
+
+- The `Raven_Autoloader` class has been removed. To install and use the
+  library you are required to use [Composer](https://getcomposer.org/).
 
-- The `Raven_Client` class has been renamed to `Client` to follow the PSR-4
-  convention.
+- The `Raven_Compat` class has been removed.
+
+- The `Raven_Util` class has been removed.
+
+- The `Raven_CurlHandler` class has been removed.
+
+- The `Raven_TransactionStack` class has been removed.
+
+- The `Raven_Exception` class has been removed.
+
+### Client
 
-- The constructor of the `Raven_Client` class has changed its signature. The only way
-  to set the DSN is now to set it in the options passed to the `Configuration`
-  class.
+- The constructor of the `Raven_Client` class has changed its signature and
```

Comment:
I don't know if we should document the new FQCN of the class because we already said somewhere before this statement that the vendor has changed and that the classes have all been renamed to follow the PSR-4 convention. Otherwise I would evaluate if we should apply this change to all other places of the document where we refer to still-existing classes even though heavily refactored

Reply:
I've already reviewed all the document in that sense, we always refer to removed classes so using the old name is fine.